lib/matplotlib/tests/test_mathtext.py

```diff
@@ -190,6 +191,24 @@ def test_mathtext_rendering(baseline_images, fontset, index, test):
              horizontalalignment='center', verticalalignment='center')
 
 
+cur_lightweight_math_tests = list(
+    filter(lambda x: x[1] is not None, enumerate(lightweight_math_tests)))
+
+
+@pytest.mark.parametrize('index, test', cur_lightweight_math_tests,
+                         ids=[str(idx) for idx, _ in cur_lightweight_math_tests])
+@pytest.mark.parametrize('fontset', ['dejavusans'])
```

Comment:
What is the point of parametrizing with a single value? Wouldn't it be more readable to hard-code such a value at it's usage location?

Reply:
The `baseline_images` fixture generates the filename using this parameter indirectly. It can't be hardcoded for this one as it's parameterized in the others.